webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarReducer.js

```diff
@@ -15,6 +15,7 @@ const initialState = Immutable({
   resourceUrl: null,
   requestError: null,
   currentPage: null,
+  search: null,
```

Comment:
do you mind change it to `searchQuery`?
I think it's more clear when your context is only the reducer.